src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -48,6 +50,37 @@ const defaultProps = {
 };
 
 function ReportActionItemCreated(props) {
+    const {windowWidth, isSmallScreenWidth} = useWindowDimensions();
+
+    const IMAGE_OFFSET_X = windowWidth / 2;
+    const IMAGE_OFFSET_Y = 75;
```

Comment:
NABish move these variables outside the function so they aren't redeclared unnecessarily.